addons/l10n_gcc_invoice_stock_account/views/report_invoice.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="arabic_english_invoice_with_snln" inherit_id="l10n_gcc_invoice.arabic_english_invoice">
+        <xpath expr="//div[@id='total']" position="after">
+            <t t-set="lot_values" t-value="o._get_invoiced_lot_values()"/>
+            <t t-if="lot_values">
+                <br/>
+                <table groups="stock_account.group_lot_on_invoice" class="table table-sm" style="width: 50%;" name="invoice_snln_table">
+                    <thead>
+                        <tr>
+                            <th><span>
+                                Product
+                                </span>
+                                <br/>
```

Comment:
@momegahed Why not use more or less the same indentation as in https://github.com/odoo/odoo/blob/bc67e6bf5520e28d2632ae700bbe1ef92e7dda6b/addons/stock_account/views/report_invoice.xml#L12 ?  (maybe you have a reason to do it like this, but it seems very vertical and less clear that way. Let me know. )

Reply:
@jco-odoo 
I prefer having Arabic text in separate lines as it is RTL and it misses up the orientation if it has English before it.
and since Arabic text is in separate line it makes since for the corresponding translations to be in separate lines also

Reply:
ok, good reason